### PR TITLE
Add `interactive-widget` property to viewport meta tag

### PR DESCRIPTION
**Pull request by bramus**
### Description

Add `interactive-widget` property to viewport meta tag

### Motivation

This new feature is available in Chrome 108. MDN has no info on it.

### Additional details

- https://developer.chrome.com/blog/viewport-resize-behavior/#opting-in-to-a-different-behavior
- https://drafts.csswg.org/css-viewport/#interactive-widget-section

### Related issues and pull requests

None.

### CODE REVIEW

File: files/en-us/web/html/viewport_meta_tag/index.md [hunk 1]
```diff
@@ -70,10 +72,29 @@ For pages that set an initial or maximum scale, this means the `width` property
...
+Interative UI widgets of the browser can influence the size of the page’s viewports. The most common such UI widget is a virtual keyboard. To control which resize behavior the browser should use, set the `interactive-widget` property.
+
+Allowed values are:
+
+- `resizes-visual`
+  - : The {{Glossary("Visual viewport")}} gets resized by the interactive widget.
+- `resizes-content`
+  - : The {{Glossary("Viewport")}} gets resized by the interactive widget.
+- `overlays-content`
+  - : Nor the {{Glossary("Viewport")}} nor the {{Glossary("Visual viewport")}} gets resized by the interactive widget.
```
Comment (sideshowbarker) [inline files/en-us/web/html/viewport_meta_tag/index.md:86]:
```suggestion
  - : Neither the {{Glossary("Viewport")}} nor the {{Glossary("Visual viewport")}} gets resized by the interactive widget.
```

File: files/en-us/web/html/viewport_meta_tag/index.md [hunk 2]
```diff
@@ -70,10 +72,29 @@ For pages that set an initial or maximum scale, this means the `width` property
...
+Allowed values are:
+
+- `resizes-visual`
+  - : The {{Glossary("Visual viewport")}} gets resized by the interactive widget.
+- `resizes-content`
+  - : The {{Glossary("Viewport")}} gets resized by the interactive widget.
+- `overlays-content`
+  - : Neither the {{Glossary("Viewport")}} nor the {{Glossary("Visual viewport")}} gets resized by the interactive widget.
+
+When the {{Glossary("Viewport")}} gets resized the initial [containing block](/en-US/docs/Web/CSS/Containing_block) also gets resized, thereby affecting the computed size of [viewport units](/en-US/docs/Web/CSS/length#viewport-percentage_lengths).
```
Comment (sideshowbarker) [inline files/en-us/web/html/viewport_meta_tag/index.md:88]:
```suggestion
When the {{Glossary("Viewport")}} gets resized, the initial [containing block](/en-US/docs/Web/CSS/Containing_block) also gets resized, thereby affecting the computed size of [viewport units](/en-US/docs/Web/CSS/length#viewport-percentage_lengths).
```

File: files/en-us/web/html/viewport_meta_tag/index.md [hunk 3]
```diff
@@ -47,6 +47,8 @@ The basic properties of the "viewport" `<meta>` tag include:
   - : Controls how much zoom in is allowed on the page. Any value less than 3 fails accessibility. Minimum: `0.1`. Maximum: `10`. Default: `10`. Negative values: ignored.
 - `user-scalable`
   - : Controls whether zoom in and zoom out actions are allowed on the page. Valid values: `0`, `1`, `yes`, or `no`. Default: `1`, which is the same as `yes`. Setting the value to `0`, which is the same as `no`, is against Web Content Accessibility Guidelines (WCAG).
+- `interactive-widget`
+  - : Specifies the effect that interactive UI widgets, such as a virtual keyboard, have on the page’s viewports. Valid values: `resizes-visual`, `resizes-content`, or `overlays-content`. Default: `resizes-visual`.
```
Comment (sideshowbarker) [inline files/en-us/web/html/viewport_meta_tag/index.md:51]:
```suggestion
  - : Specifies the effect that interactive UI widgets, such as a virtual keyboard, have on the page's viewports. Valid values: `resizes-visual`, `resizes-content`, or `overlays-content`. Default: `resizes-visual`.
```
Our house style (and markdown linter) prohibits using proper apostrophes. (Unfortunately. I’d personally prefer we could use them and weren’t forced to “fix” them like this.)

File: files/en-us/web/html/viewport_meta_tag/index.md [hunk 4]
```diff
@@ -70,10 +72,29 @@ For pages that set an initial or maximum scale, this means the `width` property
 
 Other [attributes](/en-US/docs/Web/HTML/Element/meta#attributes) that are available are `minimum-scale`, `maximum-scale`, and `user-scalable`. These properties affect the initial scale and width, as well as limiting changes in zoom level.
 
+## The effect of interactive UI widgets
+
+Interative UI widgets of the browser can influence the size of the page’s viewports. The most common such UI widget is a virtual keyboard. To control which resize behavior the browser should use, set the `interactive-widget` property.
```
Comment (sideshowbarker) [inline files/en-us/web/html/viewport_meta_tag/index.md:77]:
```suggestion
Interactive UI widgets of the browser can influence the size of the page's viewports. The most common such UI widget is a virtual keyboard. To control which resize behavior the browser should use, set the `interactive-widget` property.
```

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/html/viewport_meta_tag/index.md [hunk 5]
```diff
@@ -70,10 +72,29 @@ For pages that set an initial or maximum scale, this means the `width` property
...
+Allowed values are:
+
+- `resizes-visual`
+  - : The {{Glossary("Visual viewport")}} gets resized by the interactive widget.
+- `resizes-content`
+  - : The {{Glossary("Viewport")}} gets resized by the interactive widget.
+- `overlays-content`
+  - : Neither the {{Glossary("Viewport")}} nor the {{Glossary("Visual viewport")}} gets resized by the interactive widget.
+
+When the {{Glossary("Viewport")}} gets resized, the initial [containing block](/en-US/docs/Web/CSS/Containing_block) also gets resized, thereby affecting the computed size of [viewport units](/en-US/docs/Web/CSS/length#viewport-percentage_lengths).
```
Comment (sideshowbarker) [inline files/en-us/web/html/viewport_meta_tag/index.md:88]:
```suggestion
- `resizes-visual`
  - : The {{Glossary("visual viewport")}} gets resized by the interactive widget.
- `resizes-content`
  - : The {{Glossary("viewport")}} gets resized by the interactive widget.
- `overlays-content`
  - : Neither the {{Glossary("viewport")}} nor the {{Glossary("visual viewport")}} gets resized by the interactive widget.

When the {{Glossary("viewport")}} gets resized, the initial [containing block](/en-US/docs/Web/CSS/Containing_block) also gets resized, thereby affecting the computed size of [viewport units](/en-US/docs/Web/CSS/length#viewport-percentage_lengths).
```

**Comment by bramus** [comment]
Thanks for catching and fixing some of my rookie mistakes there. Glad to see this merged, thanks!

**Pull request merged**